plugins/package-managers/conan/src/main/kotlin/ConanV1Handler.kt

```diff
@@ -44,6 +44,12 @@ import org.ossreviewtoolkit.utils.ort.createOrtTempDir
 internal class ConanV1Handler(private val conan: Conan) : ConanVersionHandler {
     override fun getConanHome(): File = Os.userHomeDirectory.resolve(".conan")
 
+    override fun createConanProfileIfNeeded() {
+        if (!getConanHome().resolve("profiles/ort-default").isFile) {
+            conan.command.run("profile", "new", "ort-default", "--detect").requireSuccess()
```

Comment:
The "ort-default" should only be added to the script in this commit which makes use of it.